os/net/lwip/src/core/raw.c

```diff
@@ -136,29 +136,29 @@ static u8_t raw_input_match(struct raw_pcb *pcb, u8_t broadcast)
 u8_t raw_input(struct pbuf *p, struct netif *inp)
 {
 	struct raw_pcb *pcb, *prev;
-	s16_t proto;
+	/* s16_t proto; */
 	u8_t eaten = 0;
 	u8_t broadcast = ip_addr_isbroadcast(ip_current_dest_addr(), ip_current_netif());
 
 	LWIP_UNUSED_ARG(inp);
 
-#if LWIP_IPV6
-#if LWIP_IPV4
-	if (IP_HDR_GET_VERSION(p->payload) == 6)
-#endif							/* LWIP_IPV4 */
-	{
-		struct ip6_hdr *ip6hdr = (struct ip6_hdr *)p->payload;
-		proto = IP6H_NEXTH(ip6hdr);
-	}
-#if LWIP_IPV4
-	else
-#endif							/* LWIP_IPV4 */
-#endif							/* LWIP_IPV6 */
-#if LWIP_IPV4
-	{
-		proto = IPH_PROTO((struct ip_hdr *)p->payload);
-	}
-#endif							/* LWIP_IPV4 */
+/*#if LWIP_IPV6 */
+/*#if LWIP_IPV4 */
+/*	if (IP_HDR_GET_VERSION(p->payload) == 6) */
+/* #endif							/\* LWIP_IPV4 *\/ */
+/*	{ */
+/*		struct ip6_hdr *ip6hdr = (struct ip6_hdr *)p->payload; */
+/*		proto = IP6H_NEXTH(ip6hdr); */
+/*	} */
+/*#if LWIP_IPV4 */
+/*	else */
+/*#endif							/\* LWIP_IPV4 *\/ */
+/*#endif							/\* LWIP_IPV6 *\/ */
+/*#if LWIP_IPV4 */
+/*	{ */
+/*		proto = IPH_PROTO((struct ip_hdr *)p->payload); */
+/*	} */
+/*#endif							/\* LWIP_IPV4 *\/ */
```

Comment:
How about below?, rather than commenting each line.
```
#if 0
code to be commented
#endif
```

Reply:
Ok if it is more readable I'll fix it